tests/sqllogictests/suites/base/06_show/06_0001_show_create_table.test

```diff
@@ -18,20 +18,20 @@ SHOW CREATE TABLE `test`.`a` WITH QUOTED_IDENTIFIERS
 a CREATE TABLE "a" ( "a" BIGINT NOT NULL, "b" INT NOT NULL DEFAULT 3, "c" VARCHAR NOT NULL DEFAULT 'x', "d" SMALLINT NULL, "e" DATE NOT NULL ) ENGINE=NULL
 
 statement ok
-CREATE TABLE `test`.`b` ( a bigint not null, b int null default null, c varchar(255) not null, d smallint unsigned null) Engine = Null COMMENT = 'test b'
+CREATE TABLE `test`.`b` ( a bigint not null comment 'abc '' vs \' cde', b int null default null, c varchar(255) not null, d smallint unsigned null) Engine = Null COMMENT = 'abc '' vs \' cde'
```

Comment:
Add test that query from column system.tables.comment?

select
  t.comment as table_comment,
  c.comment as column_comment
from
  system.tables t
  join system.columns c on t.database = c.database
    and t.name = c.table and c.name = 'd2'
where
  t.database = 'test_schema'
  and t.name = 'comment_test'